users/permissions.py

```diff
@@ -30,26 +30,48 @@ def has_permission(self, request, view):
         if request.user.is_superuser:
             return True
 
-        if view.action == "create":
-            # only organization admins can add a user to their organization
-            user_is_organization_admin = OrganizationUser.objects.filter(
-                organization=request.data["organization"],
-                user=request.user.id,
-                role__name="org-admin",
-            ).exists()
-            return user_is_organization_admin
+        # when listing, a user should only see organization-user mapping for the organizations they have access to.
+        # this is handled by get_queryset
```

Comment:
can you please point me to where it is being handled inside get_queryset? I am not able to find anything relevant.

Reply:
This hasn't been added yet. working on this.